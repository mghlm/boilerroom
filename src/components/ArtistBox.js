import React from 'react';
import { View } from 'react-native';

const ArtistBox = (props) => {
  return (
    <View style={styles.viewStyle}>
      {props.children}
    </View>
  );
};

const styles = {
  viewStyle: {
    borderWidth: 3,
    borderColor: '#000',
    marginLeft: 5,
    marginRight: 5,
    marginTop: 10
  }
};

export default ArtistBox;
