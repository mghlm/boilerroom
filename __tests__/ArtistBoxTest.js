import 'react-native'
import React from 'react';
import ArtistBox from '../src/components/ArtistBox';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
  <ArtistBox />
).toJSON();
expect(tree).toMatchSnapshot();
});
