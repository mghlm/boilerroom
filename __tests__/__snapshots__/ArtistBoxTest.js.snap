// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#000",
      "borderColor": "#7d7d7d",
      "borderWidth": 1,
      "justifyContent": "center",
      "marginLeft": 5,
      "marginRight": 5,
      "marginTop": 10,
      "paddingBottom": 20,
      "paddingTop": 20,
    }
  }
/>
`;
